cmd/monaco/integrationtest/v2/slo-v2_integration_test.go

```diff
@@ -0,0 +1,135 @@
+//go:build integration
+
+/*
+ * @license
+ * Copyright 2025 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code-core/api"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/integrationtest"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/integrationtest/utils/monaco"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/featureflags"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/idutils"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/client"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/config"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/config/coordinate"
+	manifestloader "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/manifest/loader"
+)
+
+func TestSloV2(t *testing.T) {
+	configFolder := "test-resources/slo-v2/"
+	manifestPath := configFolder + "manifest.yaml"
+	environment := "platform_env"
+	project := "project"
+
+	// enable FF
+	t.Setenv(featureflags.ServiceLevelObjective.EnvName(), "true")
+
+	t.Run("When deploying two configs, two configs exist", func(t *testing.T) {
+		RunIntegrationWithCleanup(t, configFolder, manifestPath, environment, "SLO-V2", func(fs afero.Fs, testContext TestContext) {
+			err := monaco.RunWithFSf(fs, "monaco deploy %s --project=%s --verbose", manifestPath, project)
+			assert.NoError(t, err)
+
+			err = monaco.RunWithFSf(fs, "monaco deploy %s --project=%s --verbose", manifestPath, project)
+			assert.NoError(t, err)
+
+			sloV2Client := createSloV2Client(t, fs, manifestPath, environment)
+			result, err := sloV2Client.List(t.Context())
+			assert.NoError(t, err)
+			externalIDs := parseExternalIDs(t, result)
```

Comment:
Maybe `extractExternalIDs`? Because you don't parse the externalIDs, you parse the json response to get the externalIds